packages/core/admin/admin/src/hooks/useSettingsMenu/init.js

```diff
@@ -10,7 +10,7 @@ const init = (initialState, { settings, shouldUpdateStrapi }) => {
   // Sort the links by name
   const sortedGlobalLinks = sortLinks([...pluginsGlobalLinks, ...globalLinks]).map(link => ({
     ...link,
-    hasNotification: link.id === 'application-infos' && shouldUpdateStrapi,
+    hasNotification: link.id === '000-application-infos' && shouldUpdateStrapi,
```

Comment:
Why `000-`?

Reply:
changed made by @soupette to make sure this link always show first if I'm not mistaken!

Reply:
I would let @soupette give his opinion on it!
From what I understand we want the global links to be sorted alphabetically except for the Overview/Application page which should always be first. Currently we are sorting pluginsGlobalLinks + globalLinks with lodash's sortBy on link.id. Those 2 objects are not built in the same place (pluginsGlobalLinks is generated inside StrapiApp) 🤔

Reply:
Oh ok I understand. It's ok for me!